test/functional/apps/visualize/_visualize_listing.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import expect from 'expect.js';
+
+export default function ({ getPageObjects }) {
+  const PageObjects = getPageObjects(['visualize', 'header', 'common']);
+
+  describe('visualize listing page', function describeIndexTests() {
+    const vizName = 'Visualize Listing Test';
+
+    describe('create and delete', async function () {
+
+      before(async function () {
+        await PageObjects.visualize.gotoVisualizationLandingPage();
+        await PageObjects.visualize.checkListingSelectAllCheckbox();
+        await PageObjects.visualize.clickDeleteSelected();
+        await PageObjects.common.clickConfirmOnModal();
+      });
+
+      after(async () => {
+        // delete all
+        await PageObjects.visualize.checkListingSelectAllCheckbox();
+        await PageObjects.visualize.clickDeleteSelected();
+        await PageObjects.common.clickConfirmOnModal();
```

Comment:
We could move this to a separate test case and assert whether the vis is really gone - `expect(visCount).to.equal(0); `

Reply:
done